ui/components/src/Legend/ListLegend.tsx

```diff
@@ -31,14 +30,6 @@ export interface ListLegendProps {
  * large numbers of items when there is a single item per row.
  */
 export function ListLegend({ items, height, width, selectedItems, onLegendItemClick }: ListLegendProps) {
-  const theme = useTheme();
```

Comment:
We don't need the mock padding with the new layout that more closely matches designs. However, this change could theoretically impact consumers using this directly, so I marked this PR as a breaking change.